src/main/webapp/WEB-INF/classes/.gitignore

```diff
@@ -0,0 +1 @@
+/META-INF
```

Comment:
You should remove changes that are not related to the issue.You mentioned making changes to pass the tests, but it is unlikely that you need to do that. The test failures should be caused by other reasons.